docs/source/environments.rst

```diff
@@ -26,11 +26,13 @@ Packaged Environments
 
 The package comes with a few *default environments* which are **always available** and designed for specific schedulers.
 That includes the :py:class:`~flow.DefaultTorqueEnvironment` and the :py:class:`~flow.DefaultSlurmEnvironment`.
-This means that if you are within an environment with a *torque* or *slurm scheduler* you should be immediately able to submit to the cluster.
+This means that if you are within an environment with a *Torque* or *Slurm scheduler* you should be immediately able to submit to the cluster.
 
 In addition, **signac-flow** comes with some environments tailored to specific compute clusters that are defined in the :py:mod:`flow.environments` module.
-These environments are not automatically available.
-Instead, you need to *explictly import* the :py:mod:`flow.environments` module.
+These environments are also automatically available, but if they conflict with a specific environment of your choice, you can opt out of using these environments by setting the ``flow.IMPORT_ENVIRONMENTS`` signac config variable.
+For instance, to opt out of using these environments for a specific signac project, execute ``signac config set flow.IMPORT_ENVIRONMENTS False`` within that environment.
+This variable may also be set global signac configuration file on a given cluster.
+More generally, if you simply wish to force **signac-flow** to use a particular environment, you may do so by setting the environment variable ``SIGNAC_FLOW_ENVIRONMENT`` in your shell.
```

Comment:
It's inconsistent that we use a config value everywhere else and then a environment variable for this, but I think I'm ok with that for now. We should allow for consistent config-key environment variable name mapping as part of a larger config framework rewrite.

Reply:
I agree that it's inconsistent, but I also think that the use-cases are different enough that it might make sense. The config values are used for semi-permanent settings for a given project, and a given project may not want to import environments for some specific reason. The `SIGNAC_FLOW_ENVIRONMENT` seems like more of a testing/development feature, and not something that I would need to set for a specific project. But I agree that we should discuss this as part of a broader rewrite.